dev/devicelab/manifest.yaml

```diff
@@ -579,9 +576,6 @@ tasks:
       The test makes sure that there is no regression while renderring an image with gl on iOS.
     stage: devicelab_ios
     required_agent_capabilities: ["mac/ios", "ios/gl-render-image"]
-    #TODO(cyanglaz): The flaky flag is added because it is the first screenshot test we added.
-    # Remove the flaky flag when we are sure the test is stable.
-    flaky: true
```

Comment:
\cc @cyanglaz

Reply:
This test passed but doesn't actually WAI. I should've put a note there. It doesn't hurt to enable it tho as it will always pass.

Reply:
You sure?  I'm happy to revert these lines.

Reply:
I will send a revert PR with a revised todo message.